custom_components/waste_collection_schedule/waste_collection_schedule/source/welhat_gov_uk.py

```diff
@@ -0,0 +1,91 @@
+import requests
+from bs4 import BeautifulSoup
+import dateutil.parser as parser
+from waste_collection_schedule import Collection
+
+
+TITLE = "Welwyn Hatfield Borough Council"
+DESCRIPTION = "Source for www.welhat.gov.uk services for Welwyn Hatfield Borough Council, UK."
+URL = "https://www.welhat.gov.uk"
+TEST_CASES = {
+    "test 1 - South Red": {"uprn": "100080965745", "postcode": "AL9 5EA"},
+    "test 2 - Blue North": {"uprn": "100080977050", "postcode": "AL7 3ET"}
+}
+
+ICON_MAP = {
+    "food": "mdi:apple",
+    "garden": "mdi:tree",
+    "refuse": "mdi:trash-can",
+    "recycling": "mdi:recycle"
+}
+
+class Source:
+    def __init__(self, uprn: str, postcode: str):
+        self._uprn = uprn
+        self._postcode = postcode
+
+    def fetch(self):
+        print(self._uprn)
+        print(self._postcode)
```

Comment:
Please remove print statements

Reply:
I've created a new commit removing the print statements.